packages/desktop-client/src/components/accounts/Balance.tsx

```diff
@@ -126,34 +148,58 @@ function FilteredBalance({ filteredAmount }) {
   );
 }
 
-function MoreBalances({ balanceQuery }) {
+function MoreBalances({
+  balanceQuery,
+}: {
+  balanceQuery: ComponentProps<typeof ReconcilingMessage>['balanceQuery'];
+}) {
   const { t } = useTranslation();
 
-  const cleared = useSheetValue({
-    name: balanceQuery.name + '-cleared',
+  type SelectedBalanceClearedName = `balance-query-${string}-cleared`;
+  const cleared = useSheetValue<'balance', SelectedBalanceClearedName>({
+    name: (balanceQuery.name + '-cleared') as SelectedBalanceClearedName,
     query: balanceQuery.query.filter({ cleared: true }),
   });
-  const uncleared = useSheetValue({
-    name: balanceQuery.name + '-uncleared',
+
+  type SelectedBalanceUnclearedName = `balance-query-${string}-uncleared`;
+  const uncleared = useSheetValue<'balance', SelectedBalanceUnclearedName>({
+    name: (balanceQuery.name + '-uncleared') as SelectedBalanceUnclearedName,
     query: balanceQuery.query.filter({ cleared: false }),
   });
 
   return (
     <View style={{ flexDirection: 'row' }}>
-      <DetailedBalance name={t('Cleared total:')} balance={cleared} />
-      <DetailedBalance name={t('Uncleared total:')} balance={uncleared} />
+      <DetailedBalance
+        name={t('Cleared total:')}
+        balance={cleared}
+        isExactBalance
+      />
+      <DetailedBalance
+        name={t('Uncleared total:')}
+        balance={uncleared}
+        isExactBalance
+      />
     </View>
   );
 }
 
+type BalancesProps = {
+  balanceQuery: ComponentProps<typeof ReconcilingMessage>['balanceQuery'];
```

Comment:
This should be a `Binding`

Reply:
What makes you say that? Do you have a suggested edit?

I am seeing `balanceQuery: { name: `balance-query-${string}`; query: Query };` in a different file. 

https://github.com/actualbudget/actual/blob/339fac280605bbd344ecb3387751a9ee43ffab69/packages/desktop-client/src/components/accounts/Reconcile.tsx#L20

Reply:
This is one way I could think of to set it as a `Binding`
```
balanceQuery: Extract<
    Binding<'balance', `balance-query-${string}`>,
    { name: string; query?: Query }
  >;